### PR TITLE
fix: consistent `bindings: {code: string}`

### PR DESCRIPTION
**Pull request by mhevery**
## Description

Add a short description of:
- what changes you made, 
- why you made them, and 
- any other context that you think might be helpful for someone to better understand what is contained in this pull request. 

This sort of information is useful for people reviewing the code, as well as anyone from the future trying to understand why changes were made or why a bug started happening.

### CODE REVIEW

File: packages/core/src/parsers/builder.ts [hunk 1]
```diff
@@ -872,3 +879,20 @@ export const builderContentToMitosisComponent = (
 
   return componentJson;
 };
+function mapBuilderBindingsToMitosisBindingWithCode(
+  bindings: { [key: string]: string } | undefined,
+): MitosisNode['bindings'] {
+  const result: MitosisNode['bindings'] = {};
+  bindings &&
+    Object.keys(bindings).forEach((key) => {
+      const value: string | { code: string } = bindings[key] as any;
```
Comment (samijaber) [inline packages/core/src/parsers/builder.ts:888]: Surprised that we needed this: I thought `BuilderElement` bindings will always be `bindings: { [key: string]: string }`, so how could the value be a `{ code: string }`? Did you encounter that possibility?

**Review by samijaber — APPROVED** [review]
Types to the rescue!

File: packages/core/src/parsers/builder.ts (hunk 1 above)
Comment (samijaber) [reply packages/core/src/parsers/builder.ts:888]: If my understanding is correct, this is the only possibility here:

```suggestion
      const value: string = bindings[key]
```

Comment (mhevery) [reply packages/core/src/parsers/builder.ts:888]: Yes the tests fail without this.

**Pull request merged**